src/main/java/ee/tuleva/onboarding/comparisons/returns/provider/IndexReturnProvider.java

```diff
@@ -27,7 +27,7 @@
 
     private static final String EPI = EPIFundValueRetriever.KEY;
 
-    private static final String MARKET = WorldIndexValueRetriever.KEY;
+    private static final String MARKET = UnionStockIndexRetriever.KEY;
```

Comment:
2 problems with this:
1. It's misleading.
Previously, it was `String MARKET = "MARKET"` in short. (1:1 match between the API name and Retriever KEY)
Now, it's `String MARKET = "UNION_STOCK_INDEX"` in short. (mismatch between API name and Retriever KEY)
2. It actually changes the API. So you would need to change your clients as well. 
I.e. you cannot do `GET /returns?from=2003-01-07&keys[]=MARKET` anymore. 
You have to do `GET /returns?from=2003-01-07&keys[]=UNION_STOCK_INDEX`. 
This makes the API change backwards incompatible, which is another topic up for debate - whether we want to keep it backwards compatible or not right now.

I propose we stick to the 1:1 mapping between the API names and Retriever KEYs - and go forward with the backwards incompatible change. Why? It just keeps things simple and avoids confusion. 

The alternative would be to have "MARKET" in the API layer and "UNION_STOCK_INDEX" in the service/database layer. This is confusing since we also have a "MARKET" time series in the database.
In short, the mapping would be counter-intuitively MARKET=UNION_STOCK_INDEX not MARKET=MARKET.

Reply:
Rename `String MARKET` to `String UNION_STOCK_INDEX`.

Reply:
String MARKET changed to String UNION_STOCK_INDEX